Sateltiles/admin_user.py

```diff
@@ -0,0 +1,151 @@
+import wx
+import os
+from init import *
+import login as log
+import manage_user as manage
+import database_api as db
+
+class AdminUserPage(wx.Frame):
+    def __init__(self, parent, role, title):
+        super(AdminUserPage, self).__init__(parent, title=title, size=(600,520), style=wx.MINIMIZE_BOX| wx.SYSTEM_MENU | wx.CAPTION
+                                        | wx.CLOSE_BOX | wx.CLIP_CHILDREN)
+        wx.Frame.CenterOnScreen(self)
+        panel = wx.Panel(self)
+        self.icon = wx.Icon("iconn.ico", wx.BITMAP_TYPE_ICO)
+        self.SetIcon(self.icon)
+
+        font = wx.SystemSettings_GetFont(wx.SYS_SYSTEM_FONT)
+        font.SetPointSize(25)
+        existing_label = wx.StaticText(panel, pos=(20,20), label="Existing TMS")
+        existing_label.SetFont(font)
+
+        generate_label = wx.StaticText(panel, pos=(20,200+130), label="Generate TMS")
+        generate_label.SetFont(font)
+
+        source_filePath_label = wx.StaticText(panel, pos=(10,230+130), label="Source file (.tif) : ") 
+        self.source_filePath_entry = wx.TextCtrl(panel, pos=(100,230+130), size=(165,20)) # sorce file path entries
+
+        save_filePath_label = wx.StaticText(panel, pos=(20,260+130), label="Save location : ")
+        self.save_filePath_entry = wx.TextCtrl(panel, pos=(100,260+130), size=(165,20)) # destination file path entries
+
+        getLink_button = wx.Button(panel, label="GET TMS LINK", pos=(470,190+130), size=(100,40)) 
+        getLink_button.Bind(wx.EVT_BUTTON, self.getTMS_Link)
+
+        back_button = wx.Button(panel, label="BACK", pos=(10,310+130), size=(100,40)) # back home buttons
+        back_button.Bind(wx.EVT_BUTTON, self.backHome)
+
+        source_button = wx.Button(panel, label="Browse", pos=(280,227+130), size=(90,25)) 
+        source_button.Bind(wx.EVT_BUTTON, self.browseSourceFile)
+
+        save_button = wx.Button(panel, label="Browse", pos=(280,257+130), size=(90,25)) 
+        save_button.Bind(wx.EVT_BUTTON, self.browseSaveFile)
+
+        generate_button = wx.Button(panel, label="GENERATE", pos=(175,285+130), size=(90,30)) 
+        generate_button.Bind(wx.EVT_BUTTON, self.generateTMS)
+
+        manage_button = wx.Button(panel, label="Manage user", pos=(470,310+130), size=(100,40)) 
+        manage_button.Bind(wx.EVT_BUTTON, self.manageUser)
+
+        # database admin/information
+        database_name = "gdal2tiles"
+        database_username = "postgres"
+        database_password = "polpol01"
+        database_host = "203.159.29.196"
+        database_port = "5432"
+        try:
+            database = db.PostGresAPI(database_name, database_username, database_password, database_host, database_port) # connect to main database
+        except:
+            print("Can not connect to database (" + str(database_host) + ":" + str(database_port))

```

Comment:
string concat in python better done this way "cannot connect to database %s: %s" % (database_host, database_port)